### ISSUE TITLE
[BUG] could not fetch language 

### ISSUE THREAD

**Issue by cdobrovolny** [Usercentrics/flutter-sdk#60]
I got the following exception after start of my application. It happens only on Android devices Emulator and physical Devices:
The devices had stable internet connection
```
I/System.out( 7623): [USERCENTRICS][WARNING] Usercentrics SDK was not able to initialize online, let's try to initialize offline | cause: com.usercentrics.sdk.errors.UsercentricsException: Something went wrong while fetching the available languages.
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.facade.LanguageFacade$resolveLanguage$2.invoke(LanguageFacade.kt:24)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.facade.LanguageFacade$resolveLanguage$2.invoke(LanguageFacade.kt:23)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.async.dispatcher.DispatcherCallback.setResult$usercentrics_release(DispatcherCallback.kt:26)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.async.dispatcher.Dispatcher$dispatch$1.invokeSuspend(Dispatcher.kt:20)
I/System.out( 7623): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
I/System.out( 7623): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
I/System.out( 7623): 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
I/System.out( 7623): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
I/System.out( 7623): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
I/System.out( 7623): 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
I/System.out( 7623): Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/firebase/perf/network/FirebasePerfUrlConnection;
I/System.out( 7623): 	at com.usercentrics.sdk.services.api.http.AndroidHttpClient.createConnection(AndroidHttpClient.kt:47)
I/System.out( 7623): 	at com.usercentrics.sdk.services.api.http.AndroidHttpClient.get(AndroidHttpClient.kt:22)
I/System.out( 7623): 	at com.usercentrics.sdk.domain.api.http.HttpRequestsImpl.getSync(HttpRequestsImpl.kt:35)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.api.LanguageApi.getAvailableLanguages(LanguageApi.kt:18)
2
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.repository.LanguageRepository$fetchAvailableLanguages$response$1.invoke(LanguageRepository.kt:25)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.network.NetworkOrchestrator.resolveHttp(NetworkOrchestrator.kt:25)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.repository.LanguageRepository.fetchAvailableLanguages(LanguageRepository.kt:25)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.service.LanguageService.loadSelectedLanguage(LanguageService.kt:23)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.facade.LanguageFacade$resolveLanguage$1.invokeSuspend(LanguageFacade.kt:22)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.facade.LanguageFacade$resolveLanguage$1.invoke(Unknown Source:8)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.language.facade.LanguageFacade$resolveLanguage$1.invoke(Unknown Source:4)
I/System.out( 7623): 	at com.usercentrics.sdk.v2.async.dispatcher.Dispatcher$dispatch$1.invokeSuspend(Dispatcher.kt:18)
I/System.out( 7623): 	... 6 more
```

**usercentrics-flutter-sdk branch/commit**
usercentrics_sdk: ^2.8.0

**Comment by userCTest** [comment]
Thanks a lot for the extra info.
I guess you are still seeing the same with v2.8.1? Do you think you could provide us with a small sample app that replicates the issue?
Thanks!
Rui

**Comment by userCTest** [comment]
Thanks a lot! :)

**Comment by cdobrovolny** [comment]
Sorry for the late answer. Unfortunately I'm unable to create an example because on the brand new app it is still working. 

But maybe here is another information about the possible bug:

We included firebase_performance in the app some month ago. Now I removed this package and then the usercentrics throws this error. When i add the dependency into the project, usercentrics is working.

I Already tried to clean the cache.
